core/src/main/java/io/parsingdata/metal/expression/value/arithmetic/Mod.java

```diff
@@ -34,7 +34,7 @@ public Mod(final ValueExpression left, final ValueExpression right) {
 
     @Override
     public OptionalValue eval(final Value left, final Value right, final Environment environment, final Encoding encoding) {
-        if (right.asNumeric().compareTo(BigInteger.ZERO) < 0) {
+        if (right.asNumeric().compareTo(BigInteger.ZERO) <= 0) {
```

Comment:
Nice.

Also: I had forgotten we had actually added `Mod`, I thought it was missing.